env/scratch/kustomization.yaml

```diff
@@ -26,8 +29,9 @@ configMapGenerator:
   env: .env
 
 patches:
- - performance.yaml
- - services.yaml
+  - celery-deployment.yaml
+  - performance.yaml
+  - services.yaml
```

Comment:
Are these indented one too many, or is github diff just being dumb?

Reply:
We used to just have one space indenting here which was different from other sections, and VSCode was unhappy with it so I added another space.